### PR TITLE
docs: add English version README

### PR DESCRIPTION
**Pull request by mikaeloduh**
## Why need this change? / Root cause: 
- As titled
## Changes made:
-
## Test Scope / Change impact:
-
## Issue
- https://github.com/Game-as-a-Service/Lobby-Platform-Service/discussions/97

### CODE REVIEW

File: README-en.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Lobby-Platform-Service
+
+Game as a Service is an open source, community-based project aimed at demonstrating the best software engineering methodologies, such as Domain-Driven Design (DDD), Behavior-Driven Development (BDD), Test-Driven Development (TDD), and DevOps. You can explore the most advance software technology and how we leveraging it. Feel free to check around the code.
```
Comment (ricksu978) [inline README-en.md:3]: > Game as a Service is an open source
1. 你後面段落有引用 GaaS，所以前面這邊打全名時加個縮寫比較好 `Game as a Service (GaaS)`

> demonstrating the best software engineering methodologies
2. `the best` 這邊需要修飾一下， 可以試試看 ChatGPT 的建議

> Feel free to check around the code.
3.  ~~check around~~ check out

File: README-en.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Lobby-Platform-Service supports the backend service of GaaS, providing a gaming portal and a social community platform.
```
Comment (ricksu978) [inline README-en.md:5]: Lobby-Platform-Service ~~supports~~ is the backend service of GaaS,

**Review by ricksu978 — CHANGES_REQUESTED** [review]
Review Done.

File: README-en.md (hunk 1 above)
Comment (mikaeloduh) [reply README-en.md]: 修改在 ecc8fcf4f9356cc02d6a161320d465ce3b435295

File: README-en.md (hunk 2 above)
Comment (mikaeloduh) [reply README-en.md]: 同上

**Review by ricksu978 — APPROVED** [review]

**Pull request merged**